docs/source/development-guidelines.rst

```diff
@@ -196,6 +196,8 @@ __ https://www.sphinx-doc.org/en/master/
 
    $ pip install -r ./etc/requirements_docs.in -c ./etc/requirements_locked.txt
 
+If you would like to use Anaconda, please follow `the installation guide <https://www.zipline.io/install#managing-conda-environments>`_ utill ``source activate env_zipline``, then run the command above.
```

Comment:
I suggest we reference the docs using a relative, instead of an absolute, path. That way it works locally while developing, or hosted on another site, etc. I also updated the typo ("utill").
```suggestion
If you would like to use Anaconda, please follow :ref:`the installation guide<managing-conda-environments>` to create and activate an environment, and then run the command above.
```

I think this would require adding a label to install.rst:
```
.. _managing-conda-environments:
```